yarn.lock

```diff
@@ -87,10 +87,10 @@
     "@aws-amplify/api-graphql" "^1.2.5"
     "@aws-amplify/api-rest" "^1.2.5"
 
-"@aws-amplify/appsync-modelgen-plugin@1.22.10":
-  version "1.22.10"
-  resolved "https://registry.yarnpkg.com/@aws-amplify/appsync-modelgen-plugin/-/appsync-modelgen-plugin-1.22.10.tgz#f4ffc4543e96672c8a60152195a7cd97094084f3"
-  integrity sha512-FtncIYSeCAvOtMi45PKIB20DlpzEdW20NzTy/OGza3iNb6VPK7EEar+ZslxPOcMPK9ZSUD/eB4scK9ufKLfTCw==
+"@aws-amplify/appsync-modelgen-plugin@1.23.1":
```

Comment:
What triggered the changes to this file? It doesn't look like any dependencies were changed.

Reply:
yarn build or yarn setup-dev are the only things I can think of. 
I also followed debugger steps [here](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#debugging). I'm not sure tbh. I can revert

Reply:
I think if we don't know what caused the changes, we should back them out.

Reply:
Just tested, and everytime I do yarn setup-dev this lock file is modified. I just accidentally checked the changes in this time.

Reply:
updated